ci/cmake-multi-platform.yml

```diff
@@ -0,0 +1,60 @@
+# This starter workflow is for a CMake project running on multiple platforms. There is a different starter workflow if you just want a single platform.
+# See: https://github.com/actions/starter-workflows/blob/main/ci/cmake-single-platform.yml
+name: CMake on a multiple platforms
+
+on:
+  push:
+    branches: [ $default-branch ]
+  pull_request:
+    branches: [ $default-branch ]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      # Set fail-fast to false to ensure that feedback is delivered for all matrix combinations. Can consider changing this to true when the workflow is stable. 
```

Comment:
Should we use column-limit for the contents of the file, particularly for comments and for longer commands?

@andrewakim - Do you know if there's a GitHub Actions YAML formatting action that could just apply some standards across starter workflow file contents (and other workflows if others wanted to use it).

Reply:
Hmmm. I don't think so and I'm not sure what we enforced in the past.